torch/linalg/__init__.py

```diff
@@ -80,96 +88,97 @@
 
             [[ 1.3910,  0.0000],
             [-1.8509,  1.1014]]], dtype=torch.float64)
-    >>> torch.allclose(torch.matmul(l, l.transpose(-2, -1)), a)
+    >>> torch.allclose(l @ l.transpose(-2, -1), a)
     True
 """)
 
 inv = _add_docstr(_linalg.linalg_inv, r"""
 linalg.inv(input, *, out=None) -> Tensor
 
-Computes the multiplicative inverse matrix of a square matrix :attr:`input`, or of each square matrix in a
-batched :attr:`input`. The result satisfies the relation:
+Computes the `inverse`_ of square matrix or batch of matrices
+
+Supports :attr:`input` of float, double, cfloat and cdouble dtypes.
 
-``matmul(inv(input),input)`` = ``matmul(input,inv(input))`` = ``eye(input.shape[0]).expand_as(input)``.
+.. note:: When :attr:`input` is on a CUDA device, this function synchronizes that device with the CPU.
 
-Supports input of float, double, cfloat and cdouble data types.
+.. note:: This function is computed using LAPACK's `getrf` and `getri` routines for CPU inputs.
+          For CUDA inputs, cuSOLVER's `getrf` and `getrs` routines as well as cuBLAS' `getrf`
+          and `getri` are used if CUDA version >= 10.1.243, otherwise MAGMA's `getrf`
+          and `getri` are used instead.
 
-.. note:: When given inputs on a CUDA device, this function synchronizes that device with the CPU.
+.. seealso::
 
-.. note:: The inverse matrix is computed using LAPACK's `getrf` and `getri` routines for CPU inputs. For CUDA
-          inputs, cuSOLVER's `getrf` and `getrs` routines as well as cuBLAS' `getrf` and `getri` routines are
-          used if CUDA version >= 10.1.243, otherwise MAGMA's `getrf` and `getri` routines are used instead.
+        :func:`torch.linalg.pinv` computes the Moore-Penrose pseudoinverse of a general matrix
```

Comment:
period needed at end of this sentence.

what is a "general matrix"?

Reply:
Added periods in all the `seealso` that I missed.

I changed "general" for "of a (not necessarily square) matrix".